pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -2442,6 +2442,11 @@
       "AssemblyVersionInPackageVersion": {
         "4.0.0.0": "4.3.0"
       }
+    },
+    "System.Web.HttpUtility": {
+      "AssemblyVersionInPackageVersion": {
+        "4.0.1.0": "4.3.0"
```

Comment:
I would assume the assembly version should be 4.0.0.0 as this is the first version.